src/components/VerifyEmail.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const VerifyEmail = () => {
+  return (
+    <div>
+      <h2>Verify Email</h2>
+      <div>
+        <p>Your email has been been verified. You can now sign in with your new account.</p>
+        <a href="/">Back to Login</a>
+      </div>
+    </div>
+  );
+};
```

Comment:
Need to add functionality to actually verify the users email when they visit this page. Take a look at [this step in the documentation](https://firebase.google.com/docs/auth/custom-email-handler#:~:text=Handle%20email%20address%20verification%20by%20calling%20applyActionCode.%20For%20example%3A), I think you need to grab the `actionCode` from `EmailAction.jsx`, then call the [`applyActionCode` firebase function](https://firebase.google.com/docs/reference/js/v8/firebase.auth.Auth#applyactioncode).